src/midi/midi_mt32.cpp

```diff
@@ -626,6 +630,15 @@ bool MidiHandler_mt32::Open([[maybe_unused]] const char *conf)
 		mixer_channel->SetLowPassFilter(FilterState::Off);
 	}
 
+	// Double the baseline PCM prebuffer because MIDI is demanding
+	const auto render_ahead_ms = MIXER_GetPreBufferMs() * 2;
```

Comment:
Hmm, that seems pretty ad-hoc, but okay.

Reply:
Its roughly as lean as I could go (40 ms with default settings) where my 800 MHz stress tests were still able to avoid drop outs. Going lower, the CPU didn't have enough breathing room to digest another a big batch of MIDI/MT-32 and keep the music playing.

Reply:
Yeah, I was referring to the x2 multiplier, but okay, if it works it works :) (e.g. why not just use the prebuffer size without doubling it).